system_files/desktop/shared/usr/share/ublue-os/just/custom.just

```diff
@@ -48,6 +48,12 @@ install-nix:
 remove-nix:
   sudo /nix/nix-installer uninstall
 
+install-fleek:
+  #!/usr/bin/env bash
+  mkdir -p $HOME/.config/nix
```

Comment:
I recall $HOME not working when invoking the script with sudo -A just. I think we had to retrieve this some other way (like $SUDO_USER)

But otherwise everything here looks great. I can look more into the prior just to be sure

Reply:
Ah. Found it

```export HOME=$(getent passwd ${SUDO_USER:-$USER} | cut -d: -f6)```